tests/api/test_search.py

```diff
@@ -87,26 +86,34 @@ def test_temporal_search_open():
     assert search.end_date is None
 
 
-def test_invalid_temporal_search():
-    # Not RFC339
-    utcnow = datetime.utcnow().strftime("%Y-%m-%d")
+def test_invalid_temporal_search_date():
```

Comment:
Split the tests apart so each does a single thing, and all can be test independently.